### PR TITLE
Issue #611

### PR DESCRIPTION
**Pull request by M4ttoF**
Added button to the end of a search with no results that links to the add location page

### CODE REVIEW

File: app/resources/views/pages/location_search/spreadsheet.blade.php [hunk 1]
```diff
@@ -15,7 +15,8 @@ class="form-control" id="distance"
 			<button type="button" id="updateRadius" class="btn btn-primary">Update</button>
 	</div>
 	@if (count($locations) === 0)
-		No location found matching the specified keywords
+		<h3>No location found matching the specified keywords</h3>
+		<a class="btn btn-default" href="/locations-added-by-me">Add New Location</a>
```
Comment (joshi1983) [inline app/resources/views/pages/location_search/spreadsheet.blade.php]: Navigating directly to /add-location would be quicker for people who want to add locations.

Also, don't show this only when the user is signed in.  If they're not signed in, they won't be able to add a location which would make the link confusing.

Did you test by shrinking the width of the browser?  I didn't test this yet but that is a quick way to simulate what various mobile devices will see.  If you see anything wrong, you can update the corresponding .scss file in resources/assets/sass directory.

**Comment by M4ttoF** [comment]
@joshi1983 should be all fixed now

File: app/resources/views/pages/location_search/spreadsheet.blade.php [hunk 2]
```diff
@@ -16,7 +16,15 @@ class="form-control" id="distance"
 	</div>
 	@if (count($locations) === 0)
 		<h3>No location found matching the specified keywords</h3>
-		<a class="btn btn-default" href="/locations-added-by-me">Add New Location</a>
+			@if ($base_user->isSignedIn())
+				<a class="btn btn-default" href="/add-location">Add New Location</a>
+			@else
+				<a class="btn btn-default" href="/sign-in">Sign In</a>
+				<h4>or</h4>
+				<a class="btn btn-default" href="/add-location">Sign Up</a>
```
Comment (joshi1983) [inline app/resources/views/pages/location_search/spreadsheet.blade.php]: The href value doesn't match your "Sign Up" caption there.

File: app/resources/views/pages/location_search/spreadsheet.blade.php [hunk 3]
```diff
@@ -16,7 +16,15 @@ class="form-control" id="distance"
 	</div>
 	@if (count($locations) === 0)
 		<h3>No location found matching the specified keywords</h3>
-		<a class="btn btn-default" href="/locations-added-by-me">Add New Location</a>
+			@if ($base_user->isSignedIn())
+				<a class="btn btn-default" href="/add-location">Add New Location</a>
+			@else
+				<a class="btn btn-default" href="/sign-in">Sign In</a>
```
Comment (joshi1983) [inline app/resources/views/pages/location_search/spreadsheet.blade.php]: Did you test this link?  It looks like "/signin" is what you want without the hyphen.

File: app/resources/views/pages/location_search/spreadsheet.blade.php [hunk 4]
```diff
@@ -15,7 +15,16 @@ class="form-control" id="distance"
 			<button type="button" id="updateRadius" class="btn btn-primary">Update</button>
 	</div>
 	@if (count($locations) === 0)
-		No location found matching the specified keywords
+		<h3>No location found matching the specified keywords</h3>
+			@if ($base_user->isSignedIn())
+				<a class="btn btn-default" href="/add-location">Add New Location</a>
+			@else
+				<a class="btn btn-default" href="/sign-in">Sign In</a>
```
Comment (joshi1983) [inline app/resources/views/pages/location_search/spreadsheet.blade.php]: I think /sign-in is a broken link.  Did you test this by signing out and clicking both links?

File: app/resources/views/pages/location_search/spreadsheet.blade.php [hunk 5]
```diff
@@ -15,7 +15,16 @@ class="form-control" id="distance"
...
+				<h4>or</h4>
+				<a class="btn btn-default" href="/sign-up">Sign Up</a>
```
Comment (joshi1983) [inline app/resources/views/pages/location_search/spreadsheet.blade.php]: There is a similar problem with /sign-up.  I think it should be /signup.  Test it to be sure.

**Comment by M4ttoF** [comment]
@joshi1983 my bad, got mixed up with add-location.
I'll double check it next time

**Pull request merged**